model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -0,0 +1,188 @@
+from model_analyzer.config.input.config_defaults import DEFAULT_NUM_CONFIGS_PER_MODEL, \
+    DEFAULT_RUN_CONFIG_MIN_CONCURRENCY, DEFAULT_RUN_CONFIG_MAX_CONCURRENCY
+
+from copy import deepcopy
+
+import logging
+
+logger = logging.getLogger(LOGGER_NAME)
+
+
+class QuickPlusConcurrencySweepRunConfigGenerator(ConfigGeneratorInterface):
+    """
+    First run QuickRunConfigGenerator for a hill climbing search, then use 
+    Brute for a concurrency sweep of the default and Top N results
+    """
+
+    def __init__(self, search_config: SearchConfig,
+                 config: ConfigCommandProfile, gpus: List[GPUDevice],
+                 models: List[ConfigModelProfileSpec], client: TritonClient,
+                 result_manager: ResultManager,
+                 model_variant_name_manager: ModelVariantNameManager):
+        """
+        Parameters
+        ----------
+        search_config: SearchConfig
+            Defines parameters and dimensions for the search
+        config: ConfigCommandProfile
+            Profile configuration information
+        gpus: List of GPUDevices
+        models: List of ConfigModelProfileSpec
+            List of models to profile
+        client: TritonClient
+        result_manager: ResultManager
+            The object that handles storing and sorting the results from the perf analyzer
+        model_variant_name_manager: ModelVariantNameManager
+            Maps model variants to config names
+        
+        model_variant_name_manager: ModelVariantNameManager
+        """
+        self._search_config = search_config
+        self._config = config
+        self._gpus = gpus
+        self._models = models
+        self._client = client
+        self._result_manager = result_manager
+        self._model_variant_name_manager = model_variant_name_manager
+        self._rcg = None
+
+    def set_last_results(self, measurements):
+        self._rcg.set_last_results(measurements)
+
+    def get_configs(self) -> Generator[RunConfig, None, None]:
+        """
+        Returns
+        -------
+        RunConfig
+            The next RunConfig generated by this class
+        """
+        yield from self._execute_quick_search()
+        yield from self._sweep_concurrency_over_top_results()
+
+    def _execute_quick_search(self):
+        self._rcg = self._create_quick_run_config_generator()
+
+        yield from self._rcg.get_configs()
+
+    def _create_quick_run_config_generator(self) -> QuickRunConfigGenerator:
+        return QuickRunConfigGenerator(
+            search_config=self._search_config,
+            config=self._config,
+            gpus=self._gpus,
+            models=self._models,
+            client=self._client,
+            model_variant_name_manager=self._model_variant_name_manager)
+
+    def _sweep_concurrency_over_top_results(self):
+        top_results = self._result_manager.top_n_results(
+            n=DEFAULT_NUM_CONFIGS_PER_MODEL)
```

Comment:
This is wrong. This default is fed into the ConfigCommandProfile, and is used if not specified on the command line. You need to use the value in the config object.

Reply:
I didn't think the user had the ability to specify min/max concurrencies with a quick search, so I just figured I would use the defaults.

Reply:
This isn't min/max concurrencies. It is top-N. It should be using config.num_configs_per_model:
```
            ConfigField(
                'num_configs_per_model',
                flags=['--num-configs-per-model'],
                field_type=ConfigPrimitive(int),
                default_value=DEFAULT_NUM_CONFIGS_PER_MODEL,
                description=
                'The number of configurations to plot per model in the summary.'
            ))
```

Reply:
Understood and fixed